src/main/kotlin/no/nav/familie/klage/kabal/KabalKafkaListener.kt

```diff
@@ -99,14 +99,10 @@ data class BehandlingDetaljer(
     val ankeITrygderettenbehandlingOpprettet: AnkeITrygderettenbehandlingOpprettetDetaljer? = null,
 ) {
 
-    fun journalpostReferanser(): List<String> {
-        return klagebehandlingAvsluttet?.journalpostReferanser ?: ankebehandlingAvsluttet?.journalpostReferanser ?: listOf()
-    }
-
-    fun oppgaveTekst(): String {
-        return klagebehandlingAvsluttet?.oppgaveTekst()
-            ?: ankebehandlingOpprettet?.oppgaveTekst()
-            ?: ankebehandlingAvsluttet?.oppgaveTekst()
+    fun oppgaveTekst(behandlendeEnhet: String): String {
```

Comment:
burde vi kalle det "saksbehandlersEnhet" ? For å ikke mikse begrepene vi allerede har brukt om "behandlendeEnhet" ? 🤔

Reply:
enig i det